src/Mod/Spreadsheet/Gui/DlgSettings.ui

```diff
@@ -26,6 +26,45 @@
       <string>Import/Export Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
+      <item row="3" column="2">
+       <widget class="Gui::PrefLineEdit" name="formatString">
+        <property name="text">
+            <string  notr="true">%V = %A</string>
+        </property>
+        <property name="placeholderText">
+            <string>%V = %A</string>
+        </property>
+        <property name="toolTip">
+         <string>The format of the alias string presentation.
+Defaults to: %V = %A
+
+%A - alias name
+%V - cell value</string>
+        </property>
+        <property name="prefEntry" stdset="0">
+         <cstring>ImportExportFormatString</cstring>
```

Comment:
??? What's this name ?

Reply:
oups, too much copy past going here ! 
changed to DisplayAliasFormatString